src/common/GitHubVS.ruleset

```diff
@@ -37,6 +37,7 @@
         <Rule Id="CA1501" Action="None" />
         <Rule Id="CA1502" Action="None" />
         <Rule Id="CA1506" Action="None" />
+        <Rule Id="CA1701" Action="None" />
```

Comment:
I should have linked that. https://msdn.microsoft.com/en-us/library/ms182187.aspx?f=255&MSPPError=-2147217396

> CA1303: Do not pass literals as localized parameters
> A method passes a string literal as a parameter to a constructor or method in the .NET Framework class library and that string should be localizable.
> This warning is raised when a literal string is passed as a value to a parameter or property and one or more of the following cases is true: